src/main/java/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -734,4 +735,28 @@ public static String formatActivityEventId(Set<String> activityEventIds, String
         }
         return id;
     }
+    
+    /**
+     * Select the member of the list that matches the ISO 639 alpha-2 or alpha-3 language code,
+     * or else the member with the English ("en") value. If neither exists, returns null.
+     */
+    public static <T extends HasLang> T selectByLang(List<T> items, List<String> languages, T defaultValue) {
+        checkNotNull(items);
+        
+        if (languages == null) {
+            languages = ImmutableList.of();
+        }
+        T englishItem = null;
+        for (T item : items) {
+            for (String lang : languages) {
+                if (lang.equalsIgnoreCase(item.getLang())) {
+                    return item;
+                }
+            }
+            if ("en".equalsIgnoreCase(item.getLang())) {
+                englishItem = item;
```

Comment:
So if my Langs were 'es' and 'de' and the items in order were 'de' and 'es', would this select es?

Reply:
Added this as a test and adjusted code so it works.